fastlane/Jetpack-Fastfile

```diff
@@ -166,7 +231,18 @@ private_lane :jetpack_appstore_code_signing do |options|
     type: "appstore",
     team_id: get_required_env("EXT_EXPORT_TEAM_ID"),
     readonly: true,
-    app_identifier: "com.automattic.jetpack"
+    app_identifier: APP_IDENTIFIER
   )
 end
 
+def read_version_from_config
+  fastlane_require 'Xcodeproj'
+  version = ''
+  # If the file is not available, the method will raise so we should be fine
+  # not handling that case. We'll never return an empty string.
+  File.open(File.join(PROJECT_ROOT_FOLDER, 'Config', 'Version.public.xcconfig')) do |config|
+    configuration = Xcodeproj::Config.new(config)
+    version = configuration.attributes['VERSION_SHORT']
+  end
+  return version
```

Comment:
I _think_ that `File.open` is able to return whatever the block we pass to returns?

Which means we could get rid of the intermediate variable and use this instead:
```suggestion
  # If the file is not available, the method will raise so we should be fine
  # not handling that case. We'll never return an empty string.
  File.open(File.join(PROJECT_ROOT_FOLDER, 'Config', 'Version.public.xcconfig')) do |config|
    configuration = Xcodeproj::Config.new(config)
    configuration.attributes['VERSION_SHORT']
  end
```
With the return being implicit in ruby (returning the return value of the last statement, here the `File.open`, which should itself implicitly return the return value of the last statement of its block, here `configuration.attributes['VERSION_SHORT')`)

Worth double-checking it works tho, since I haven't tested this and only mention this from memory.

Reply:
Thank you for the suggestion @AliSoftware 🙇 

I just tried that for an upcoming PR and it works 👌